README.md

```diff
@@ -26,15 +28,16 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+In practice it doesn't make any difference but data-ng-click is better usage for html validation purposes.
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+"A" is the best usage of "ng-app" because "ng-app" is the app's module name and usually should be contain all of the code via <html> tag.
+
 ```
 
```

Comment:
Correct, the main idea is to avoid having multiple angular apps on the same page.